Gemfile.lock

```diff
@@ -54,6 +55,10 @@ GEM
       unicode-display_width (~> 1.0, >= 1.0.1)
     ruby-progressbar (1.8.1)
     sass (3.4.22)
```

Comment:
> The docs we link to are linked to a gem version, so I guess it's implicit (if not necessarily true from our perspective)?

Where's that? I couldn't find any reference to a specific sass version.

I'll make it arturoable.